### PR TITLE
Asoragna/humble

### PR DESCRIPTION
**Pull request by alsora**
## Description

Update the simulator to ROS 2 Humble

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Test in progress.

  - verified that build is successfull
 - verified that I can start the simulator

### CODE REVIEW

File: irobot_create_ignition/irobot_create_ignition_bringup/package.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <exec_depend>irobot_create_ignition_plugins</exec_depend>
 
   <!-- ROS IGN -->
-  <exec_depend>ign_ros2_control</exec_depend>
+  <exec_depend>gazebo_ros2_control</exec_depend>
```
Comment (justinIRBT) [inline irobot_create_ignition/irobot_create_ignition_bringup/package.xml:21]: Is this the right thing to do long term or a work around?

**Review by justinIRBT — APPROVED** [review]

File: irobot_create_ignition/irobot_create_ignition_bringup/package.xml (hunk 1 above)
Comment (alsora) [reply irobot_create_ignition/irobot_create_ignition_bringup/package.xml:21]: I think that this package has been renamed from galactic to humble, now that "gazebo classic" is deprecated.

Comment (alsora) [reply irobot_create_ignition/irobot_create_ignition_bringup/package.xml:21]: Actually I was wrong.
The repository has been renamed and I didn't find a package named `ign_ros2_control` in Humble (I only found `gazebo_ros2_control`) so I made that assumption.

The truth is that this package has not been released yet for Humble.

**Comment by alsora** [comment]
I tested the Humble branch:
 - with gazebo classic it works out of the box
 - with gazebo ignition it first require to build from sources https://github.com/ros-controls/gz_ros2_control

I added a disclaimer in the README.md and opened a ticket https://github.com/ros-controls/gz_ros2_control/issues/82

**Pull request merged**